README.md

```diff
@@ -1,16 +1,18 @@
 # rmf_obstacle_detectors
+
```

Comment:
I'd prefer not adding a newline after each heading. Same thing for other changes below where a newline is added before a codeblock.

Reply:
oops, the auto formatter from work was on. I have fixed them as required. Thanks!